bolts/data/datasets/vision/isic.py

```diff
@@ -229,7 +229,7 @@ def _remove_uncertain_diagnoses(metadata_df: pd.DataFrame) -> pd.DataFrame:
             metadata_df["meta.clinical.benign_malignant"].isin({"benign", "malignant"})
         ]  # throw out unknowns
         malignant_mask = labels_df["meta.clinical.benign_malignant"] == "malignant"
-        labels_df["malignant"] = malignant_mask.astype(np.uint8)
+        labels_df["malignant"] = malignant_mask.astype("int8")
```

Comment:
Do you have a good reason for this? I like to avoid string representations of things wherever possible - they're not safe.

Reply:
I don't have a great reason. I agree that `np.uint8` is better, but [in the pandas docs they provide the numpy type as a string](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.astype.html), so I assume that this is the "correct" way to do it. But 🤷‍♂️ .

The type should at least be `uint8` though, not `int8` as I've put.

Reply:
Let's keep it as it as np.uint8 then it your only reason is the pandas docs - we follow our own philosophy.